server/conf/ephemeral.conf

```diff
@@ -0,0 +1,269 @@
+samantha.ephemeral {
+  engineType = "RECOMMENDER"
+
+  entityDaosConfig {
+    entityDaoKey = "entityDaoName"
+    RequestEntityDAO {
+      daoConfigClass = "org.grouplens.samantha.server.dao.RequestEntityDAOConfig"
+      entitiesKey = "entities"
+    }
+    RequestItemIDListDAO {
+      daoConfigClass = "org.grouplens.samantha.server.dao.RequestItemIDListDAOConfig"
+      attrName = "movieId"
+      itemListKey = "movieIds"
+    }
+    CSVFileEntityDAO {
```

Comment:
for ValueSampledDAO, implement a new type of Indexer depending on a CSVFileIndexer